### PR TITLE
chore: add SENDGRID_API_KEY to backend secrets

### PR DESCRIPTION
**Pull request by christellevs**
Related to  / https://github.com/ubio/squad-nodescript/issues/667

### CODE REVIEW

File: secrets/apps/production/backend-secrets.yaml
Comment (dani-beltran) [inline secrets/apps/production/backend-secrets.yaml:28]: I have no idea, what is this `mac` secret for ?

**Review by dani-beltran — APPROVED** [review]
Looks good 🍏 
I left a question, but is for me to learn.

File: secrets/apps/production/backend-secrets.yaml
Comment (inca) [reply secrets/apps/production/backend-secrets.yaml:28]: These are generated by `sops`. If you haven't seen/used those before, I suggest taking a brief look here https://github.com/ubio/infrastructure/blob/main/README.md and here https://github.com/ubio/engineering/blob/main/docs/guides/deploying-apps.md#secrets

**Review by inca — APPROVED** [review]
Btw don't wait for approval of things like these, I'm happy if you leave those for post-reviews

**Pull request merged**
